### ISSUE TITLE
Enable jsx-a11y

### ISSUE THREAD

**Issue by JRIngram** [JRIngram/leitner#57]
# Feature Request - *Enable jsx-a11y*
## Describe the feature

jsx-a11y is current in the dev dependencies but isn't turned on. This should be turned on in the `client/` by:
* Add `plugin:jsx-a11y/recommended` in the extends section of `.eslintrc.json`
* Add `jsx-a11y` to the plugins section of `.eslintrc.json`

Any issues found by turning the new linter rules on should be fixed as part of a PR for this feature.
